src/pages/steps/AdditionalInformationStep/AdditionalInformationStep.tsx

```diff
@@ -1,3 +1,4 @@
+import { map, mapValues } from 'lodash';
```

Comment:
```suggestion
import { mapValues } from 'lodash';
```

I don't think this import is used